tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -416,7 +416,7 @@ def start_cmd(self, node):
         if self.enable_autocommit:
             cmd += " --enable-autocommit "
 
-        cmd += " --reset-policy %s --group-id %s --topic %s --broker-list %s --session-timeout %s" % \
+        cmd += " --reset-policy %s --group-id %s --topic %s --bootstrap-server %s --session-timeout %s" % \
```

Comment:
We should continue using `--broker-list` if the node version is lower than 4.0.

Reply:
Updated it. Thanks for the review.